data/darktableconfig.xml.in

```diff
@@ -200,6 +200,38 @@
     <shortdescription>enable disk backend for full preview cache</shortdescription>
     <longdescription>if enabled, write full preview to disk (.cache/darktable/) when evicted from the memory cache.\nnote that this can take a lot of memory (several gigabytes for 20k images) and will never delete cached thumbnails again.\nit's safe though to delete these manually, if you want.\nlight table performance will be increased greatly when zooming image in full preview mode.</longdescription>
   </dtconfig>
+  <dtconfig prefs="processing" section="cpugpu" restart="true">
+    <name>backthumbs_mipsize</name>
+    <type>
+      <enum>
+        <option>never</option>
```

Comment:
Default is disabled so safe regarding your point 2.

Reply:
No restarting necessary any more as we can do everything at runtime. Also we have the "reinitialize" option now.